core/src/cluster_nodes.rs

```diff
@@ -237,14 +239,24 @@ impl ClusterNodes<RetransmitStage> {
             .position(|node| node.pubkey() == self.pubkey)
             .unwrap();
         if drop_redundant_turbine_path(shred.slot(), root_bank) {
+            let root_distance = match self_index {
+                0 => 0,
+                1..=200 => 1,
```

Comment:
I think we should be using the `DATA_PLANE_FANOUT` constant here

Reply:
done